diff --git WORKSPACE WORKSPACE
index cb125e13..208e735c 100644
--- WORKSPACE
+++ WORKSPACE
@@ -45,7 +45,7 @@ ENVOY_SHA256 = "7d0aa580e12aab470617ed4affebe56190716c1801568861f64d1918f4f6c07b

 ENVOY_ORG = "envoyproxy"

-ENVOY_REPO = "istio"
+ENVOY_REPO = "envoy"

 # To override with local envoy, just pass `--override_repository=envoy=/PATH/TO/ENVOY` to Bazel or
 # persist the option in `user.bazelrc`.
