diff --git .bazelrc .bazelrc
index 364b1971..5359bb90 100644
--- .bazelrc
+++ .bazelrc
@@ -21,7 +21,7 @@ build:remote --remote_timeout=7200
 # ========================================
 
 # Enable libc++ and C++20 by default.
-build:linux --config=clang
+#build:linux --config=clang
 
 # put /usr/local/bin before /usr/bin to avoid picking up wrong python3.6 when building envoy.tls.key_providers.cryptomb
 build:linux --action_env=PATH=/usr/local/bin:/bin:/usr/bin
@@ -56,13 +56,17 @@ build:release-symbol -c opt
 build:release-symbol -c opt
 
 # Debug builds
-build:debug -c dbg
+#build:debug -c dbg
 
 # Add compile option for all C++ files
 # TODO(kuat) re-enable after fixing upstream dd/tracing-lib.
 # build --cxxopt -Wnon-virtual-dtor
 build --cxxopt -Wformat
 build --cxxopt -Wformat-security
+build --cxxopt -Wno-range-loop-analysis
+build --cxxopt --no-warnings
+build --cxxopt -Wno-missing-field-initializers
+build --cxxopt -w
 
 # CI sanitizer configuration
 #
diff --git WORKSPACE WORKSPACE
index 0108e5d2..0d762f3b 100644
--- WORKSPACE
+++ WORKSPACE
@@ -90,5 +90,5 @@
 envoy_toolchains()
 
 load("@llvm_toolchain//:toolchains.bzl", "llvm_register_toolchains")
 
-llvm_register_toolchains()
+# llvm_register_toolchains()
