diff --git Makefile.core.mk Makefile.core.mk
index c8c79f35..38d5bce9 100644
--- Makefile.core.mk
+++ Makefile.core.mk
@@ -38,14 +38,14 @@ BAZEL_STARTUP_ARGS := --client_debug $(BAZEL_STARTUP_ARGS)
 BAZEL_BUILD_ARGS := -s --sandbox_debug --verbose_failures $(BAZEL_BUILD_ARGS)
 endif
 
-ifeq "$(origin WITH_LIBCXX)" "undefined"
-WITH_LIBCXX := $(shell ($(CXX) --version | grep ^g++ >/dev/null && echo 0) || echo 1)
-endif
-ifeq "$(WITH_LIBCXX)" "1"
-BAZEL_CONFIG = --config=libc++
-else
+#ifeq "$(origin WITH_LIBCXX)" "undefined"
+#WITH_LIBCXX := $(shell ($(CXX) --version | grep ^g++ >/dev/null && echo 0) || echo 1)
+#endif
+#ifeq "$(WITH_LIBCXX)" "1"
+#BAZEL_CONFIG = --config=libc++
+#else
 BAZEL_CONFIG =
-endif
+#endif
 
 UNAME := $(shell uname)
 ifeq ($(UNAME),Linux)

